http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/JsonJacksonPayloadIT.java

```diff
@@ -15,7 +15,8 @@ public class JsonJacksonPayloadIT extends JsonPayloadIT {
     private static final String REST_JACKSON = "rest-jackson";
 
     @QuarkusApplication(classes = { FootballTeamResource.class, FootballTeam.class, Person.class,
-            PersonResource.class }, properties = "oidcdisable.properties", dependencies = @Dependency(artifactId = "quarkus-"
+            PersonResource.class,
+            JsonExceptionMapper.class }, properties = "oidcdisable.properties", dependencies = @Dependency(artifactId = "quarkus-"
```

Comment:
Looking at this and you add `JsonExceptionMapper` but that class implements `ExceptionMapper<JsonbException>` which stating `JsonbException` but this should test Jackson. So to me it's seems that there is always thrown only `JsonbException`

Reply:
Current code just use JsonB, then I'll resolve this comment.